### PR TITLE
add mitoz-3.0a0

### PR DESCRIPTION
**Pull request by linzhi2013**
Describe your pull request here

Trying to ADD MitoZ (https://github.com/linzhi2013/MitoZ) to bioconda, thanks!

MitoZ: A toolkit for assembly, annotation, and visualization of animal mitochondrial genomes (https://doi.org/10.1093/nar/gkz173)

----

Please read the [guidelines for Bioconda recipes](https://bioconda.github.io/contributor/guidelines.html) before opening a pull request (PR).

* If this PR adds or updates a recipe, use "Add" or "Update" appropriately as the first word in its title.
* New recipes not directly relevant to the biological sciences need to be submitted to the [conda-forge channel](https://conda-forge.org/docs/) instead of Bioconda.
* PRs require reviews prior to being merged. Once your PR is passing tests and ready to be merged, please issue the `@BiocondaBot please add label` command.
* Please post questions [on Gitter](https://gitter.im/bioconda/Lobby) or ping `@bioconda/core` in a comment.

<details>
  <summary>Please use the following BiocondaBot commands:</summary>

Everyone has access to the following BiocondaBot commands, which can be given in a comment:

<table>
  <tr>
    <td><code>@BiocondaBot please update</code></td>
    <td>Merge the master branch into a PR.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please add label</code></td>
    <td>Add the <code>please review & merge</code> label.</td>
  </tr>
  <tr>
    <td><code>@BiocondaBot please fetch artifacts</code></td>
    <td>Post links to CI-built packages/containers. <br />You can use this to test packages locally.</td>
  </tr>
</table>

For members of the Bioconda project, the following command is also available:

<table>
  <tr>
    <td><code>@BiocondaBot please merge</code></td>
    <td>Upload built packages/containers and merge a PR. <br />Someone must approve a PR first! <br />This reduces CI build time by reusing built artifacts.</td>
  </tr>
</table>

Also, the bot watches for comments from non-members that include `@bioconda/<team>` and will automatically re-post them to notify the addressed `<team>`.

</details>

### CODE REVIEW

**Comment by linzhi2013** [comment]
@BiocondaBot please add label

**Comment by dpryan79** [comment]
@BiocondaBot please fetch artifacts

File: recipes/mitoz/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    - mitoz = mitoz.MitoZ:main
+
+source:
+  url: https://github.com/linzhi2013/MitoZ/files/8565237/mitoz-3.0a0.tar.gz
+  # url: /home/gmeng/dev/MitoZ_private/dist/mitoz-{{ version }}.tar.gz
+  md5: 5c8afdde1890f0905e4819a7d97b9b8d
+
+requirements:
+  host:
+    - python=3.9
```
Comment (dpryan79) [inline recipes/mitoz/meta.yaml:21]: Is it really correct that only python 3.9 is supported?

File: recipes/mitoz/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,50 @@
+{% set version = "3.0a0" %}
```
Comment (dpryan79) [inline recipes/mitoz/meta.yaml:1]: We try to avoid alpha releases, since they're not intended for use except by the developers.

Comment (linzhi2013) [reply recipes/mitoz/meta.yaml:1]: Thanks! I follow your suggestion and further update the software.

Now it falls back to Python 3.6 and updates to 3.0.

I have created a new pull request.

**Pull request closed without merging**